src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1463,60 +1614,141 @@ components:
       required:
         - offenderIds
 
-    CiagInductionSummaryListResponse:
-      title: CiagInductionSummaryListResponse
+    CreateInductionRequest:
+      title: CreateInductionRequest
+      description: A request to create an Induction.
       type: object
-      description: Contains a list of CIAG Induction summaries. Migrated from hmpps-ciag-careers-induction-api.
       properties:
-        ciagProfileList:
-          type: array
-          description: A List of prisoners' CIAG Induction summaries. Can be empty but not null.
-          items:
-            $ref: '#/components/schemas/CiagInductionSummaryResponse'
+        workOnRelease:
+          $ref: '#/components/schemas/CreateWorkOnReleaseRequest'
+        previousQualifications:
+          $ref: '#/components/schemas/CreatePreviousQualificationsRequest'
+        previousTraining:
+          $ref: '#/components/schemas/CreatePreviousTrainingRequest'
+        previousWorkExperiences:
+          $ref: '#/components/schemas/CreatePreviousWorkExperiencesRequest'
+        inPrisonInterests:
+          $ref: '#/components/schemas/CreateInPrisonInterestsRequest'
+        personalSkillsAndInterests:
+          $ref: '#/components/schemas/CreatePersonalSkillsAndInterestsRequest'
+        futureWorkInterests:
+          $ref: '#/components/schemas/CreateFutureWorkInterestsRequest'
       required:
-        - ciagProfileList
-
-    CiagInductionSummaryResponse:
-      title: CiagInductionSummaryResponse
+        - workOnRelease
```

Comment:
I'm making this the only mandatory field. I've considered making `previousTraining` mandatory too, as I think that's always asked (and `NONE` is a viable option), but I don't want to make decisions based on the current screens. 

Stepping back from the screen flow, it makes sense that a prisoner is always asked if they hope to work  - it feels like a key part of an induction - but gathering other apsects, such as their previous training, does not seem vital.